src/web/mjs/connectors/Alphapolis.mjs

```diff
@@ -0,0 +1,63 @@
+
+    async _getPages(chapter) {
+        const script = `
+            var _0x5012=['WOrFCSkq','W4NdUfK=','WP1BFSkhWO0=','W78rE3vjW5lcHW==','W4tcU8kEWOlcUG==','W5vyvrddMxOTW6pcKG==','w0ldKCogeLVdGmkijCkedSkIDCkUwSk2WPxcQmoKWOjRjCkCWRCSgGm='];(function(_0x1225c6,_0x5012b3){var _0x5e03e4=function(_0x138c07){while(--_0x138c07){_0x1225c6['push'](_0x1225c6['shift']());}};_0x5e03e4(++_0x5012b3);}(_0x5012,0x11a));var _0x5e03=function(_0x1225c6,_0x5012b3){_0x1225c6=_0x1225c6-0x0;var _0x5e03e4=_0x5012[_0x1225c6];if(_0x5e03['eKfTmD']===undefined){var _0x138c07=function(_0x57a40b){var _0x350a78='abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789+/=',_0x32282b=String(_0x57a40b)['replace'](/=+$/,'');var _0x4f83c6='';for(var _0x52bde4=0x0,_0x3eec1b,_0xbdcf31,_0x52be32=0x0;_0xbdcf31=_0x32282b['charAt'](_0x52be32++);~_0xbdcf31&&(_0x3eec1b=_0x52bde4%0x4?_0x3eec1b*0x40+_0xbdcf31:_0xbdcf31,_0x52bde4++%0x4)?_0x4f83c6+=String['fromCharCode'](0xff&_0x3eec1b>>(-0x2*_0x52bde4&0x6)):0x0){_0xbdcf31=_0x350a78['indexOf'](_0xbdcf31);}return _0x4f83c6;};var _0x48deab=function(_0x58d5e0,_0x479064){var _0x2b7425=[],_0x1e8dd5=0x0,_0x436c7a,_0x103faf='',_0x289431='';_0x58d5e0=_0x138c07(_0x58d5e0);for(var _0x471ec8=0x0,_0x1c67fe=_0x58d5e0['length'];_0x471ec8<_0x1c67fe;_0x471ec8++){_0x289431+='%'+('00'+_0x58d5e0['charCodeAt'](_0x471ec8)['toString'](0x10))['slice'](-0x2);}_0x58d5e0=decodeURIComponent(_0x289431);var _0x2dc407;for(_0x2dc407=0x0;_0x2dc407<0x100;_0x2dc407++){_0x2b7425[_0x2dc407]=_0x2dc407;}for(_0x2dc407=0x0;_0x2dc407<0x100;_0x2dc407++){_0x1e8dd5=(_0x1e8dd5+_0x2b7425[_0x2dc407]+_0x479064['charCodeAt'](_0x2dc407%_0x479064['length']))%0x100,_0x436c7a=_0x2b7425[_0x2dc407],_0x2b7425[_0x2dc407]=_0x2b7425[_0x1e8dd5],_0x2b7425[_0x1e8dd5]=_0x436c7a;}_0x2dc407=0x0,_0x1e8dd5=0x0;for(var _0x5f4041=0x0;_0x5f4041<_0x58d5e0['length'];_0x5f4041++){_0x2dc407=(_0x2dc407+0x1)%0x100,_0x1e8dd5=(_0x1e8dd5+_0x2b7425[_0x2dc407])%0x100,_0x436c7a=_0x2b7425[_0x2dc407],_0x2b7425[_0x2dc407]=_0x2b7425[_0x1e8dd5],_0x2b7425[_0x1e8dd5]=_0x436c7a,_0x103faf+=String['fromCharCode'](_0x58d5e0['charCodeAt'](_0x5f4041)^_0x2b7425[(_0x2b7425[_0x2dc407]+_0x2b7425[_0x1e8dd5])%0x100]);}return _0x103faf;};_0x5e03['YyIaTO']=_0x48deab,_0x5e03['hTxvhs']={},_0x5e03['eKfTmD']=!![];}var _0x309aea=_0x5e03['hTxvhs'][_0x1225c6];return _0x309aea===undefined?(_0x5e03['qLtoxU']===undefined&&(_0x5e03['qLtoxU']=!![]),_0x5e03e4=_0x5e03['YyIaTO'](_0x5e03e4,_0x5012b3),_0x5e03['hTxvhs'][_0x1225c6]=_0x5e03e4):_0x5e03e4=_0x309aea,_0x5e03e4;};var _0x34e597=_0x5e03;new Promise(_0x138c07=>_0x138c07($(_0x34e597('0x4','3O%j'))[_0x34e597('0x5','OenM')]()[_0x34e597('0x0','OenM')](/push.['"](http[^'"]+)['"]/g)[_0x34e597('0x6','yqs9')](_0x309aea=>_0x309aea[_0x34e597('0x2','z6Dy')]('\x22')[0x1][_0x34e597('0x1','WmhG')](/[0-9]+x[0-9]+/,_0x34e597('0x3','3chS')))));
+        `;
+        const request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        return await Engine.Request.fetchUI(request, script);
```

Comment:
No need to use await, a `Promise` is fully compatible with async, in fact async function promisifies the result:
"_If the result is a promise, return it, if not wrap the result in a promise and return it_"